lib/rails-jquery-autocomplete/orm/active_record.rb

```diff
@@ -26,11 +26,20 @@ def active_record_get_autocomplete_items(parameters)
 
         scopes.each { |scope| items = items.send(scope) } unless scopes.empty?
 
-        items = items.select(get_autocomplete_select_clause(model, method, options)) unless options[:full_model]
+        select_method = options[:select_method] || :select
+
+        if select_method == :select
+          items = items.select(get_autocomplete_select_clause(model, method, options)) unless options[:full_model]
+        end
         items = items.where(get_autocomplete_where_clause(model, term, method, options)).
             limit(limit).order(order)
         items = items.where(where) unless where.blank?
-
+        if select_method == :pluck
+          columns = get_autocomplete_select_clause(model, method, options)
+          items = items.pluck(*columns)
+          columns = columns.map{|column| column.sub(/^#{model.table_name}\./, '')}
```

Comment:
Line is too long. [82/80]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.